Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
+		return "newspaperArticle";
+}
+
+function doWeb(doc, url) {
+	var arts = new Array();
+	scrape(doc);
+}
+
+function scrape(doc) {
+	var newArticle = new Zotero.Item('newspaperArticle');
+	newArticle.url = doc.location.href;
+	newArticle.title = ZU.trimInternal(ZU.xpathText(doc, '//div[@class = "dv3-clanek-left-fix"]/h1')).replace(/^,/, "");
+	var date = ZU.xpathText(doc, '(//div[@class="dv3-clanek-datum"])');
+	var matched = ZU.trimInternal(date).match(/dnes/)
+	if (matched) {
+		var actualDate = new Date();
+		var actualDateStr = '';
+		actualDateStr = actualDate.getUTCFullYear() + '-' +
+    	('00' + (actualDate.getUTCMonth()+1)).slice(-2) + '-' +
+    	('00' + actualDate.getUTCDate()).slice(-2);
+    	date = actualDateStr;
+	} else {
+		var parts = ZU.trimInternal(date).split('.');
+		
+		var actualDate = new Date(parts[2],parts[1]-1, parts[0]);
+		var actualDateStr = '';
+		actualDateStr = actualDate.getUTCFullYear() + '-' +
+    	('00' + (actualDate.getUTCMonth()+1)).slice(-2) + '-' +
+    	('00' + actualDate.getDate()).slice(-2);
```

Comment:
I would just reuse the date that is displayed instead of converting it to UTC.
`actualDate = parts[2] + '-' + ZU.lpad(parts[1], 2) + '-' + ZU.lpad(parts[0], 2);`